### PR TITLE
P2PW-1589 - Adaptive amount text size in send and swap

### PR DESCRIPTION
**Pull request by kuznetsov**

### CODE REVIEW

**Comment by gslevinkov** [comment]
wrong commit naming

File: app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapFragment.kt [hunk 1]
```diff
@@ -77,40 +81,14 @@ class OrcaSwapFragment :
     private var onBackPressedCallback: OnBackPressedCallback? = null
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        Timber.d("onViewCreated()")
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapFragment.kt:84]: no need for this I think

File: app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapFragment.kt [hunk 2]
```diff
@@ -149,6 +127,51 @@ class OrcaSwapFragment :
         presenter.loadInitialData()
     }
 
+    private fun setupViews() = with(binding) {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapFragment.kt:130]: thx for extracting all this to a separate method!

File: app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapFragment.kt [hunk 3]
```diff
@@ -149,6 +127,51 @@ class OrcaSwapFragment :
...
+        }
+        sourceImageView.setOnClickListener { presenter.loadTokensForSourceSelection() }
+        destinationImageView.setOnClickListener { presenter.loadTokensForDestinationSelection() }
+        destinationTextView.setOnClickListener { presenter.loadTokensForDestinationSelection() }
+        availableTextView.setOnClickListener { presenter.calculateAvailableAmount() }
+        maxTextView.setOnClickListener { presenter.calculateAvailableAmount() }
+
+        setupAmountListener()
+
+        exchangeImageView.setOnClickListener { presenter.reverseTokens() }
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapFragment.kt:147]: maybe format all setOn*Listener in one style?

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapFragment.kt (hunk 3 above)
Comment (kuznetsov) [reply app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapFragment.kt:147]: We can consider it

File: app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapFragment.kt (hunk 1 above)
Comment (kuznetsov) [reply app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapFragment.kt]: Tanks

**Review by kabramovp2p — APPROVED** [review]

**Review by 1sakov — APPROVED** [review]

**Pull request merged**